beacon-chain/core/validators/validator.go

```diff
@@ -11,25 +11,10 @@ import (
 	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
 	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
 	"github.com/prysmaticlabs/prysm/shared/bytesutil"
-	"github.com/prysmaticlabs/prysm/shared/hashutil"
 	"github.com/prysmaticlabs/prysm/shared/params"
 	"github.com/prysmaticlabs/prysm/shared/sliceutil"
 )
 
-// GenesisValidatorRegistry creates a new validator set that is used to
-// generate a new bootstrapped state.
-func GenesisValidatorRegistry() []*pb.Validator {
```

Comment:
This function was only used in tests, so it is now irrelevant. Instead, setupInitialDeposits in the tests was revamped to handle RANDAO.